src/components/SearchBar/SearchBar.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react'
+import styles from "./SearchBar.module.css";
+import {Button} from "react-bootstrap";
+import icon from "./Vector.svg";
```

Comment:
there's no ./Vector.svg! if you need the svg, put it in the public folder and import it from there